docs/data-guidelines.md

```diff
@@ -380,3 +380,50 @@ For example, `HTMLMediaElement`'s `play()` method returns a promise, recorded li
 ```
 
 This guideline is based on discussion in [#11630](https://github.com/mdn/browser-compat-data/pull/11630).
+
+## Global APIs
+
+An API is considered global when it is available in at least two global scopes. Such APIs are recorded in the `api/_globals/` folder.
+
+These are the web's global scopes at the time of writing:
+
+- [`Window`](https://developer.mozilla.org/en-US/docs/Web/API/Window)
+- [Workers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API): [`DedicatedWorkerGlobalScope`](https://developer.mozilla.org/en-US/docs/Web/API/DedicatedWorkerGlobalScope), [`SharedWorkerGlobalScope`](https://developer.mozilla.org/en-US/docs/Web/API/SharedWorkerGlobalScope), [`ServiceWorkerGlobalScope`](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerGlobalScope) (these inherit from the abstract [`WorkerGlobalScope`](https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope))
+- [Worklets](https://developer.mozilla.org/en-US/docs/Web/API/Worklet): `AnimationWorkletGlobalScope`, [`AudioWorkletGlobalScope`](https://developer.mozilla.org/en-US/docs/Web/API/AudioWorkletGlobalScope), `LayoutWorkletGlobalScope`, `PaintWorkletGlobalScope` (these inherit from the abstract `WorkletGlobalScope`)
+
+For example, the [`fetch()`](https://developer.mozilla.org/en-US/docs/Web/API/fetch) method is global because it is available in `Window` and `WorkerGlobalScope`, recorded like this in `api/_globals/fetch.json`:
+
+```json
+{
+  "api": {
+    "fetch": {
+      "__compat": {},
+      "worker_support": {
+        "__compat": {
+          "description": "Available in workers",
+          "support": {}
+        }
+      }
+    }
+  }
+}
+```
+
+Note that all APIs defined on the [`WindowOrWorkerGlobalScope`](https://html.spec.whatwg.org/multipage/webappapis.html#windoworworkerglobalscope-mixin) mixin are considered global.
+
+These APIs are also considered global although they're not available in _all_ workers:
+
+- The `cookieStore` property, available in `Window` and `ServiceWorkerGlobalScope`.
+- The `requestAnimationFrame()` function, available in `Window` and `DedicatedWorkerGlobalScope`.
+
+TODO: should we really still use "worker_support" here, which makes it look like these are available in all workers?
```

Comment:
I think the answer for this should be the same as for interfaces that use `[Exposed=(Window,DedicatedWorker)]`, which is the case for `MediaSource`, for example. I don't know what I think the answer there should be, but lean towards just using "worker_support" since that is the only precedent.